tutorials/advanced_features/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -1,36 +1,49 @@
 {
```

Comment:
<ul><li>"... we can create a new <code>SimConfig</code> object <strong>and assign </strong>it to the..." </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>